media-sound/pnmixer/pnmixer-0.7.1.ebuild

```diff
@@ -11,7 +11,7 @@ SRC_URI="https://github.com/nicklan/pnmixer/archive/v${PV/_rc/-rc}.tar.gz -> ${P
 
 LICENSE="GPL-3"
 SLOT="0"
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="~amd64 ~x86 ~ppc"
```

Comment:
Please keep keywords sorted in `ekeyword`-ish way. Preferably just use `ekeyword` tool to update keywords.

Reply:
Thanks for your feedback! Hope I got that right now. Had some help on IRC. ;) I will of course take your advice in accout for future PRs.